### PR TITLE
add 08-16-2017 minutes

### PR DESCRIPTION
**Pull request by Trott**
Closes: https://github.com/nodejs/CTC/issues/162

### CODE REVIEW

**Review by mhdawson — APPROVED** [review]
LGTM

File: meetings/2017-08-16.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Node Foundation CTC Meeting 2017-08-16
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=YtrD_RZa_sg
+* **GitHub Issue**: [CTC#140](https://github.com/nodejs/CTC/issues/162)
```
Comment (azu) [inline meetings/2017-08-16.md]: `CTC#140` is `CTC#162`?

Comment (Trott) [reply meetings/2017-08-16.md]: Fixed, thanks.

File: meetings/2017-08-16.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# Node Foundation CTC Meeting 2017-08-16
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=YtrD_RZa_sg
+* **GitHub Issue**: [CTC#162](https://github.com/nodejs/CTC/issues/162)
+* **Minutes Google Doc**: <https://docs.google.com/document/d/1W690JFykwVpN_V8zCSs_6yDTnFqAgE3juRz5k28SMWg>
```
Comment (targos) [inline meetings/2017-08-16.md]: https://docs.google.com/document/d/1dpfHhEyR0dj-9Wua_3m0_ycrsNTCa_fHB1n8krGctnc

File: meetings/2017-08-16.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+* _Previous Minutes Google Doc: <https://docs.google.com/document/d/1ZoLJImAGLJxivjLyCqzl7qXpjTOts1cKLwiR2XnzOLA>_
```
Comment (targos) [inline meetings/2017-08-16.md]: https://docs.google.com/document/d/1fwVahOUo7ilQKgXU7343Z18GXyX5M0GwARuguoFYvtE

**Review by targos — APPROVED** [review]
lgtm with links fixed

**Pull request merged**